docs/03-tutorials/00-observability/obsv-02-create-and-configure-grafana-dashboard.md

```diff
@@ -12,76 +12,46 @@ Follow these sections to create the Gauge dashboard type for the `cpu_temperatur
+3. For your new query, select **Prometheus** from the data source selector.
 
-   ![Create a dashboard](./assets/create-dashboard.png)
+4. Pick the `cpu_temperature_celsius` metric.
 
-3. Select **Add new panel**.
+5. To to retrieve the latest metric value on demand, activate the **Instant** switch.
 
-   ![Add Query](./assets/add-query.png)
+6. From the visualization panels, select the **Gauge** dashboard type.
 
-4. Select Prometheus data source from the **Queries to** drop-down list and pick the `cpu_temperature_celsius` metric.
-
-   ![New dashboard](./assets/new-dashboard.png)
-
-5. Toggle the **Instant** query to be able to retrieve the latest metric value on demand.
-
-   ![Instant option](./assets/instant.png)
-
-6. Switch on the right menu to the **Panel** tab and select in the **Visualization** section the **Gauge** dashboard type.
-
-   ![Gauge dashboard type](./assets/gauge-dashboard-type.png)
-
-7. Click the save button in the top right corner of the page to save the changes. Provide a name for the dashboard.
-
-   ![Save the dashboard](./assets/save-dashboard.png)
+7. Save your changes and provide a name for the dashboard.
 
 ## Configure the dashboard
 
 1. To edit the dashboard settings, go to the **Panel Title** options and select **Edit**.
 
-   ![Edit the dashboard](./assets/edit-dashboard.png)
-
-2. Back in the right menu switch to the **Field** tab and go to the **Standard options** section, set up the measuring unit to Celsius degrees to reflect the metric data type.
-
-   ![Temperature](./assets/temperature-celsius.png)
+2. Find the **Field** tab and set the measuring unit to Celsius degrees to reflect the metric data type.
 
 3. Set the minimum metric value to `60` and the maximum value to `90` to reflect the `cpu_temperature_celsius` metric value range.
 
-   ![Minimum and maximum values](./assets/min-max-values.png)
-
-4. Set a red color threshold to `75` for the dashboard to turn red once the CPU temperature reaches and exceeds this value.
-
-   ![Threshold](./assets/threshold.png)
-
-5. Go to the **Panel** tab and give a title to the dashboard.
+4. For the dashboard to turn red once the CPU temperature reaches and exceeds 75 °C, set a red color threshold to `75`.
 
-   ![Panel title](./assets/panel-title.png)
+5. Go to the **Panel** tab and give a title to the dashboard - for example, `CPU Temperature`.
 
-6. Enable the **Labels** option to display this range on the dashboard.
+6. Under **Panel > Display**, make sure the threshold labels and threshold markers are activated to display this range on the dashboard.
 
-   ![Panel title](./assets/enable-labels.png)
-
-7. Click the save button in the top right corner of the page to save the changes. Add an optional note to describe the changes made.
-
-   ![Note](./assets/save-note.png)
+7. Save your changes. We recommend that you add a note to describe the changes made.
 
 ## Verify the dashboard
```

Comment:
```suggestion
## Verify the changes
```
or `Verify the new settings` or `Check the dashboard`? You're not really verifying that the dashboard exists, but rather that the changes have actually been saved and the new settings are being applied.

Reply:
You're absolutely right. I'll try "Check the dashboard".